languages/csharp/exercises/concept/classes/.docs/instructions.md

```diff
@@ -0,0 +1,81 @@
+In this exercise you'll be playing around with a remote controlled car, that you've finally saved enough money for to buy.
+
+Each time you drive the car using the remote control, it covers 20 meters and drains one percent of the battery (which starts at 100%).
+
+The remote controlled car has a fancy LED display that shows two bits of information:
+
+- The total distance it has driven, displayed as: `"Driven <METERS> meters"`.
+- The remaining battery charge, displayed as: `"Battery at <PERCENTAGE>%"`.
+
+If the battery is at 0%, you can't drive the car anymore and the battery display will show `"Battery empty"`.
+
+You have six tasks, each of which will work with remote controller car instances.
+
+### 1. Buy a brand-new remote controller car
+
+Implement the `RemoteControlCar.Buy()` method to return a brand-new remote controlled car instance:
+
+```csharp
+RemoteControlCar car = RemoteControlCar.Buy();
+```
+
+### 2. Display the distance driven
+
+Implement the `RemoteControlCar.DistanceDisplay()` method to return the distance as displayed on the LED display:
```

Comment:
```suggestion
Implement the `DistanceDisplay()` method to return the distance as displayed on the LED display:
```

I find it so confusing that both class and instance methods use the same structure in these descriptions. Maybe we could standardise so if it's an instance method, we don't show the class name before it?

Reply:
Hmmm, I see. What if we would do:

> Implement the `DistanceDisplay()` method to return the distance as displayed on the LED display:

vs.

> Implement the static `DistanceDisplay()` method to return the distance as displayed on the LED display:

To make static the exception?

Reply:
Yeah, maybe make static bold (* *)?

Reply:
That could work. Let me update